app/servers/main/logic/js/server.js

```diff
@@ -146,3 +146,10 @@ function savePage() {
 
 function evictPage() {
 }
+
+window.onunload =  function() {
+    frequencyDialer.cleanup();
+    eventsAPI.disconnect();
+    historyAPI.disconnect();
+    favoritesAPI.disconnect();
+};
```

Comment:
Wrong indent.

Also if you start the unloading by disconnecting client first, not sure you need to rely on the unload event.